### ISSUE TITLE
[Crash] Crash when blocking app permissions and returning to the app

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#9364]
## Steps to reproduce
1. In Fenix go to Settings>Site permissions, open one of the options.
2. Either open System settings>Apps>Fenix>Permissions and *block a permission*. 
3. Either do this by pressing the "Go to settings" button from the app, on a permission blocked by Android. Change some permissions.
4. Return to the app.

Note that this only happens if a permission is turned off.

### Expected behavior
Permissions updated, no crash.
### Actual behavior
Crash.
[permissions change crash.txt](https://github.com/mozilla-mobile/fenix/files/4376088/permissions.change.crash.txt)

### Device information

* Android device: Nexus 9 (Android 7.1.1), HTC Desire 820 (Android 6.0.1)
* Fenix version: Beta 4.2. It is also reproducible on beta 4.1.0

**Comment by Amejia481** [comment]
Looking through the logs, Fenix is trying access a property that is null.

```
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'android.view.View android.view.ViewStub.inflate()' on a null object reference
        at org.mozilla.fenix.HomeActivity.getSupportActionBarAndInflateIfNecessary(HomeActivity.kt:258)
        at org.mozilla.fenix.ext.FragmentKt.showToolbar(Fragment.kt:44)
        at org.mozilla.fenix.settings.sitepermissions.SitePermissionsManagePhoneFeatureFragment.onCreate(SitePermissionsManagePhoneFeatureFragment.kt:61)
```

**Comment by ekager** [comment]
@Amejia481 I think we can just move the `showToolbar(phoneFeature.getLabel(requireContext()))` to the end of `onResume` to resolve this?

**Comment by Amejia481** [comment]
Thanks that fix the issue :)
I opened this pr for it https://github.com/mozilla-mobile/fenix/pull/9378

**Comment by Amejia481** [comment]
@sv-ohorvath would you mind re-testing?

**Comment by sv-ohorvath** [comment]
Verified as fixed on the new beta build (4.2.0-beta2).
Thanks!
